Description: Downgrading serde_yaml dep from 0.9 to 0.8.
Author: Igor Petruk ipetruk@google.com
Last-Update: 2023-01-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: scriptisto/Cargo.toml
===================================================================
--- scriptisto.orig/Cargo.toml
+++ scriptisto/Cargo.toml
@@ -230,7 +230,7 @@ version = "1.0"
 version = "1.0"
 
 [dependencies.serde_yaml]
-version = "0.9"
+version = "0.8"
 
 [dependencies.users]
 version = "0.11"
